censys/cli/commands/view.py

```diff
@@ -87,7 +86,7 @@ def include(parent_parser: argparse._SubParsersAction, parents: dict):
         "-o",
         "--output",
         type=str,
-        help="json output file path",
+        help="json output file path",  # ?? is this always json?
```

Comment:
Would we be able to remove these comments?

Reply:
yes